reference/tax_settings.v3.yml

```diff
@@ -178,6 +190,18 @@ components:
           default: true
           type: boolean
           description: This setting applies only if a merchant enters tax-inclusive prices. When enabled, the store subtracts the itemʼs store tax rate before calculating tax using the shopperʼs tax zone. The tax-exclusive amount will be the same across all tax zones. When disabled, the tax-inclusive price remains the same across all tax zones; only the tax amount will vary based on the shopperʼs location. The tax-exclusive amount may vary among tax zones. These calculations are relevant for tax pricing and tax quotations that use basic tax.
+        should_use_geolocation_to_determine_guest_shopper_tax_zone:
```

Comment:
🍻 Seems a bit funny that we need to describe this twice - is there a better approach?

i.e. `Tax_Settings` and `Tax_Settings_Req` would surely be a match for the majority of the time?

Might be something for @bc-andreadao to advise on.

Reply:
Good point, I removed `Tax_Settings_Req` 👍